names.adoc

```diff
@@ -2,183 +2,235 @@
 |Jakarta Mail
 |{gitBaseURL}/javamail[Jakarta Mail]
 |mail
+|mail
 
 |{projectBaseURL}/ee4j.jms[ee4j.jms]
 |Jakarta Messaging
 |{gitBaseURL}/jms-api[Jakarta Messaging]
 |messaging
+|messaging
 
 |{projectBaseURL}/ee4j.jpa[ee4j.jpa]
 |Jakarta Persistence
 |{gitBaseURL}/jpa-api[Jakarta Persistence]
 |persistence
+|persistence
 
 .3+|{projectBaseURL}/ee4j.jakartaee-platform[ee4j.jakartaee-platform]
 .3+|Jakarta EE Platform
 |{gitBaseURL}/jakartaee-platform[Jakarta Managed Beans]
 |managed-beans
+|N/A
 
 |{gitBaseURL}/jakartaee-platform[Jakarta EE Full Profile]
 |full-profile
+| N/A
 
 |{gitBaseURL}/jakartaee-platform[Jakarta EE Web Profile]
 |web-profile
+| N/A
 
 |{projectBaseURL}/ee4j.jaxrs[ee4j.jaxrs]
 |Jakarta RESTful Web Services
 |{gitBaseURL}/jaxrs-api[Jakarta RESTful Web Services]
 |restful-ws
+|ws.rs
 
 |{projectBaseURL}/ee4j.es[ee4j.es]
 |Jakarta Security
 |{gitBaseURL}/security-api[Jakarta Security]
 |security
+|security
 
 |{projectBaseURL}/ee4j.faces[ee4j.faces]
 |Jakarta Server Faces
 |{gitBaseURL}/mojarra[Jakarta Server Faces]
 |faces
+|N/A
 
 .2+|{projectBaseURL}/ee4j.jsp[ee4j.jsp]
-.2+|Jakarta Server Pagers
-|{gitBaseURL}/jsp-api[Jakarta Server Pagers]
+.2+|Jakarta Server Pages
+|{gitBaseURL}/jsp-api[Jakarta Server Pages]
 |pages
+|servlet.jsp
 
 |[Jakarta Debugging Support for Other Languages]
 |debugging
+| N/A
 
 |{projectBaseURL}/ee4j.servlet[ee4j.servlet]
 |Jakarta Servlet
 |{gitBaseURL}/servlet-api[Jakarta Servlet]
 |servlet
-
+|servlet
 
 .5+|{projectBaseURL}/ee4j.jakartaee-stable[ee4j.jakartaee-stable]
 .5+|Jakarta Stable APIs
 |{gitBaseURL}/enterprise-deployment[Jakarta Deployment]
 |deployment
+|N/A
 
 |{gitBaseURL}/jws-api[Jakarta Enterprise Web Services]
 |enterprise-ws
+|N/A
 
 |{gitBaseURL}/management-api[Jakarta Management]
 |management
+|N/A
 
 |{gitBaseURL}/jaxr-api[Jakarta XML Registries]
 |xml-registries
+|N/A
 
 |{gitBaseURL}/jax-rpc-api[Jakarta XML RPC]
 |xml-rpc
+|N/A
 
 |{projectBaseURL}/ee4j.jstl[ee4j.jstl]
 |Jakarta Standard Tag Library
 |{gitBaseURL}/jstl-api[Jakarta Standard Tag Library]
 |tags
+|servlet.jsp.jstl
 
 |{projectBaseURL}/ee4j.jta[ee4j.jta]
 |Jakarta Transactions
 |{gitBaseURL}/jta-api[Jakarta Transactions]
 |transactions
+|transaction
 
-|{projectBaseURL}/ee4j.websocket[ee4j.websocket]
-|Jakarta WebSocket
+.2+|{projectBaseURL}/ee4j.websocket[ee4j.websocket]
+.2+|Jakarta WebSocket
 |{gitBaseURL}/websocket-api[Jakarta WebSocket]
 |websocket
+|websocket
+
+|{gitBaseURL}/websocket-api[Jakarta WebSocket]
+|websocket-client
+|websocket.client
 
 |{projectBaseURL}/ee4j.jaxb[ee4j.jaxb]
 |Jakarta XML Binding
 |{gitBaseURL}/jaxb-api[Jakarta XML Binding]
 |xml-binding
+|xml.bind
 
 .3+|{projectBaseURL}/ee4j.jaxws[ee4j.jaxws]
 .3+|Jakarta XML Web Services
 |{gitBaseURL}/saaj-api[Jakarta SOAP Attachments]
 |soap
+|xml.soap
 
 |{gitBaseURL}/jax-ws-api[Jakarta XML Web Services]
 |xml-ws
+|xml.ws
 
 |{gitBaseURL}/jws-api[Jakarta Web Services Metadata]
 |ws-metadata
+|N/A
```

Comment:
@starksm64 Wouldn't this be 'jws'?

https://jakarta.ee/specifications/web-services-metadata/3.0/apidocs/jakarta.jws/module-summary.html

Reply:
It should be; jakarta.jws:jakarta-jws-api:3.0 has a module-info declaring 'jakarta.jws'.

Reply:
I thought for EE10 metadata was being folded into Jakarta XML Web Services 4.0, at leas that is what is in the https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan, so I put N/A. I'll double check with @lukasj

Reply:
Jakarta EE 9* uses module name `jakarta.jws`
for EE `10+`, the API is part of xml ws spec, so the module name is `jakarta.xml.ws`